api/src/main/java/org/apache/cloudstack/api/command/user/network/CreateNetworkCmd.java

```diff
@@ -218,7 +218,10 @@ public String getNetworkName() {
     }
 
     public String getDisplayText() {
-        return displayText;
+        if(displayText == null)
+            return name;
+        else
+            return displayText;
```

Comment:
```suggestion
        return StringUtils.isEmpty(displayText) ? name : displayText;
```

Reply:
Made the changes  as suggested.

Reply:
I am unresolving this @rahulbcn27 . I don't see the `==` replaced by `StringUtils.isEmpty(..)`

Reply:
Fixed as suggested